WalletWasabi.Gui/Controls/LockScreen/PinLockScreenViewModel.cs

```diff
@@ -54,8 +54,7 @@ public PinLockScreenViewModel() : base()
 				});
 
 			this.WhenAnyValue(x => x.PinInput)
-				.Select(Guard.Correct)
-				.Where(x => x.Length != 0)
+				.Where(x => !string.IsNullOrEmpty(x) && x.Length != 0)
```

Comment:
Isn't the second close redundant, meaning if `!string.IsNullOrEmpty(x)` is true then for sure `x.Length != 0` is true.

Reply:
You have right, I clear the `x.Length != 0` one. Thanks! 👍

Reply:
It is still there, maybe you fixed it locally but you didn't push.

Reply:
I know, I was waiting for @nopara73 answer but maybe he is busy. I will push it.